chain/types/mock/chain.go

```diff
@@ -49,6 +49,11 @@ func MkBlock(parents *types.TipSet, weightInc uint64, ticketNonce uint64) *types
 		panic(err)
 	}
 
+	pstateRoot := c
+	if parents != nil {
+		pstateRoot = parents.Blocks()[0].ParentStateRoot
```

Comment:
this lets us use this to cheaply make chains that are valid enough for testing out chainstore stuff.